changelog/6065.feature.rst

```diff
@@ -0,0 +1 @@
+Add support for rule data in YAML format.
```

Comment:
should we add this or wait until we got docs and everything?

Reply:
I think it's helpful for now, but we can always consolidate related changelogs later